fe/fe-core/src/main/java/com/starrocks/sql/optimizer/cost/CostModel.java

```diff
@@ -194,13 +194,12 @@ public CostEstimate visitPhysicalDistribution(PhysicalDistributionOperator node,
                     result = CostEstimate.ofCpu(statistics.getOutputSize(outputColumns));
                     break;
                 case BROADCAST:
-                    int parallelExecInstanceNum = Math.max(1, getParallelExecInstanceNum(context));
```

Comment:
Non-pipeline should still use `parallelExecInstanceNum`. 

How about retaining this invocation like follows.
```java
        private int getParallelExecInstanceNum(ExpressionContext context) {
            if (ConnectContext.get().getSessionVariable().enablePipeline()) {
                return 1;
            }
            return Math.min(ConnectContext.get().getSessionVariable().getDegreeOfParallelism(),
                    context.getRootProperty().getLeftMostScanTabletsNum());
        }
```

Reply:
Non-pipeline rather not depends on `getDegreeOfParallelism`?

Reply:
But  `parallelExecInstanceNum` should be considered for non-pipeline?